### ISSUE TITLE
Installing problems in Python 3.9.12 Numpy 1.21.5

### ISSUE THREAD

**Issue by IgnacioRubioScola** [claudiodsf/stockwell#7]
Hi, 
I cannot install Stockwell in Linux with Python 3.9.12 Numpy 1.21.5.
To reproduce the error, download the code, extract and then
```
 xxx@xxx: pip install .
Processing ~/Downloads/stockwell-master
  DEPRECATION: A future pip version will change local packages to be built in-place without first copying to a temporary directory. We recommend you use --use-feature=in-tree-build to test your packages with this new behavior before it becomes the default.
   pip 21.3 will remove support for this functionality. You can find discussion regarding this at https://github.com/pypa/pip/issues/7555.
  Installing build dependencies ... done
  Getting requirements to build wheel ... done
    Preparing wheel metadata ... done
Requirement already satisfied: numpy>=1.18 in ~/miniconda3/lib/python3.9/site-packages (from stockwell==0+unknown) (1.21.5)
Building wheels for collected packages: stockwell
  Building wheel for stockwell (PEP 517) ... done
  Created wheel for stockwell: filename=stockwell-0+unknown-cp39-cp39-linux_x86_64.whl size=30531 sha256=3668038edf9336ccb613e1dbfd7071e10b09a673d1800140ca5f2b3de2e25e00
  Stored in directory: ~/.cache/pip/wheels/90/b6/62/b90786979ab2e9d77c3e53be01e275394e6f616accf04d0535
Successfully built stockwell
Installing collected packages: stockwell
Successfully installed stockwell-0+unknown

xxx@xxx:~/Downloads/stockwell-master$ python
Python 3.9.12 (main, Apr  5 2022, 06:56:58) 
[GCC 7.5.0] :: Anaconda, Inc. on linux
Type "help", "copyright", "credits" or "license" for more information.    
>>> from stockwell import st
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "~/Downloads/stockwell-master/stockwell/__init__.py", line 2, in <module>
    from .lib import st  #NOQA
ImportError: cannot import name 'st' from 'stockwell.lib' (~/Downloads/stockwell-master/stockwell/lib/__init__.py)
```
Thank you!
Regards,
Ignacio

PS: I have the same problem using `pip install stockwell`

**Comment by claudiodsf** [comment]
Dear Ignacio,

try to run `python` from a different directory than `stockwell-master`.

**Comment by IgnacioRubioScola** [comment]
I got the following error:
```
xxx@xxx:~$ python
Python 3.9.12 (main, Apr  5 2022, 06:56:58) 
[GCC 7.5.0] :: Anaconda, Inc. on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import stockwell
RuntimeError: module compiled against API version 0xf but this version of numpy is 0xe
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "~/miniconda3/lib/python3.9/site-packages/stockwell/__init__.py", line 2, in <module>
    from .lib import st  #NOQA
ImportError: numpy.core.multiarray failed to import
```

**Comment by claudiodsf** [comment]
What's your numpy version?

`python -c "import numpy; print(numpy.version.version)"`

**Comment by IgnacioRubioScola** [comment]
```
xxx@xxx:~$ python -c "import numpy; print(numpy.version.version)"
1.21.5
```

**Comment by claudiodsf** [comment]
I think that the you have a compilation problem. The error says:

    RuntimeError: module compiled against API version 0xf but this version of numpy is 0xe

So it seems that the version of NumPy you use to compile is not the same that you use when you launch Python and import `stockwell`.

Have you tried by installing a precompiled package through `pip install stockwell` and then launching `python` from a different directory than `stockwell-master`?

**Comment by IgnacioRubioScola** [comment]
```
xxx@xxx:~$ pip uninstall stockwell
Found existing installation: stockwell 0+unknown
Uninstalling stockwell-0+unknown:
  Would remove:
    ~/miniconda3/lib/python3.9/site-packages/stockwell-0+unknown.dist-info/*
    ~/miniconda3/lib/python3.9/site-packages/stockwell/*
Proceed (Y/n)? y
  Successfully uninstalled stockwell-0+unknown

xxx@xxx:~$ pip install stockwell
Collecting stockwell
  Using cached stockwell-1.0.4-cp39-cp39-linux_x86_64.whl
Requirement already satisfied: numpy>=1.18 in ./miniconda3/lib/python3.9/site-packages (from stockwell) (1.21.5)
Installing collected packages: stockwell
Successfully installed stockwell-1.0.4

xxx@xxx:~$ python
Python 3.9.12 (main, Apr  5 2022, 06:56:58) 
[GCC 7.5.0] :: Anaconda, Inc. on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import stockwell
RuntimeError: module compiled against API version 0xf but this version of numpy is 0xe
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "~/miniconda3/lib/python3.9/site-packages/stockwell/__init__.py", line 2, in <module>
    from .lib import st  #NOQA
ImportError: numpy.core.multiarray failed to import

```

**Comment by IgnacioRubioScola** [comment]
I solved the problem after updating to Numpy 1.22.3

**Comment by claudiodsf** [comment]
Great! Closing this issue, then.
